swapfactory/swap_factory_test.go

```diff
@@ -422,3 +423,39 @@ func TestSwap_MultipleSwaps(t *testing.T) {
 		require.Equal(t, StageCompleted, stage)
 	}
 }
+
+func TestSwapFactory_MultVerifyForClaim(t *testing.T) {
+	auth, conn, _ := setupXMRTakerAuth(t)
+	defer conn.Close() // remove when merging into update-ganache branch
+
+	_, _, contract, err := DeploySwapFactory(auth, conn)
+	require.NoError(t, err)
+	// WaitForReceipt goes here when merging into update-ganache branch
+
+	dleq := &dleq.CGODLEq{}
```

Comment:
My IDE doesn't like it when we name variables using the same name as an imported package. Would it be a good thing if I made a future pull request that removes the `CGODLEq` object  and we would just call `Prove` and `Verify` directly on the package name?